### PR TITLE
Content moderator data plane swagger

### PR DESCRIPTION
**Pull request by alvadb**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by vishrutshah — CHANGES_REQUESTED** [review]
@alvadb Let's make sure that [CI](https://travis-ci.org/Azure/azure-rest-api-specs/builds/264511255?utm_source=github_status&utm_medium=notification) is green first and also let me know if you have any questions. 

Also, curious to know whether you are doing this on behalf of service team?

Thanks!

File: specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
+            "name": "cacheimage",
+            "in": "query",
+            "description": "Use cached image.",
+            "type": "boolean"
+          },
+          {
+            "$ref": "#/parameters/ocpApimSubscriptionKeyParameter"
+          },
+          {
+            "$ref": "#/parameters/ocpApimSubscriptionKeyParameter"
```
Comment (vishrutshah) [inline specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json]: CI seems to be failing due to the duplicate parameters. 

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/264921311#L3501

**Review by vishrutshah — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json [hunk 2]
```diff
@@ -0,0 +1,591 @@
...
+          },
+          {
+            "$ref": "#/parameters/contentTypeHeader2"
+          },
+          {
+            "name": "file",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "file"
```
Comment (vishrutshah) [inline specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json]: This seems to be non-confimant to the swagger 2.0 schema. Please use 

```json5
"schema" : {
  "type": "object",
  "format": "file"
}
```

Reference: https://github.com/Azure/azure-rest-api-specs/pull/1544

File: specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json [hunk 3]
```diff
@@ -0,0 +1,591 @@
...
+          },
+          {
+            "$ref": "#/parameters/contentTypeHeader3"
+          },
+          {
+            "name": "file",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "file"
```
Comment (vishrutshah) [inline specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json]: Here as well

File: specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json [hunk 4]
```diff
@@ -0,0 +1,591 @@
...
+          },
+          {
+            "$ref": "#/parameters/contentTypeHeader4"
+          },
+          {
+            "name": "file",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "file"
```
Comment (vishrutshah) [inline specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json]: here as well

File: specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json [hunk 5]
```diff
@@ -0,0 +1,591 @@
...
+          },
+          {
+            "$ref": "#/parameters/contentTypeHeader5"
+          },
+          {
+            "name": "file",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "file"
```
Comment (vishrutshah) [inline specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json]: here as well

**Review by vishrutshah — CHANGES_REQUESTED** [review]

**Review by vishrutshah — CHANGES_REQUESTED** [review]
Please review the syntax failures on CI https://travis-ci.org/Azure/azure-rest-api-specs/jobs/267346600#L3521 Thanks!

**Comment by veronicagg** [comment]
@alvadb are there updates on this one?

**Comment by salameer** [comment]
@alvab what are the next steps here on this PR?

**Comment by vishrutshah** [comment]
Closing this as service team is working on improvements in their fork. Once completed, they'll send a new PR for review. Thanks@

**Pull request closed without merging**